libr/core/cmd_anal.c

```diff
@@ -4748,14 +4748,14 @@ static void _anal_calls(RCore *core, ut64 addr, ut64 addr_end) {
 				if (!anal_is_bad_call (core, from, to, addr, buf, bufi)) {
 					fcn = r_anal_get_fcn_in (core->anal, op.jump, R_ANAL_FCN_TYPE_ROOT);
 					if (!fcn) {
-						r_core_anal_fcn (core, op.jump, addr, R_ANAL_REF_TYPE_NULL, depth);
```

Comment:
RCore.anal_fcn is having reftype after from/to, so make things consistent, revert the change in the xrefs api